src/mscorlib/src/System/Reflection/Assembly.CoreCLR.cs

```diff
@@ -52,7 +52,20 @@ private static Assembly LoadFromResolveHandler(object sender, ResolveEventArgs a
             string requestedAssemblyPath = Path.Combine(Path.GetDirectoryName(requestorPath), requestedAssemblyName.Name+".dll");
 
             // Load the dependency via LoadFrom so that it goes through the same path of being in the LoadFrom list.
-            return Assembly.LoadFrom(requestedAssemblyPath);
+            Assembly resolvedAssembly = null;
+
+            try
+            {
+                resolvedAssembly = Assembly.LoadFrom(requestedAssemblyPath);
+            }
+            catch(Exception)
```

Comment:
I thought about doing so but the event callback expects a null for any scenario that does not return a valid assembly. Hence, I added a generic catch.